### DISCUSSION TITLE
Flatten members

### DISCUSSION THREAD

**Discussion by miloush** [icsharpcode/ILSpy#3080] [General] [answered]
Am I misremembering that, or was there an option to show all inherited memberes on a type? (or I just cannot find it)

**Comment by siegfriedpammer** [comment, marked as answer]
We have the option to find base and derived types, but not members.
There is also an analyzer that looks for members that override or implement a member, but I am not sure, if that's what you are looking for.

**Reply by miloush** [reply]
Thanks @siegfriedpammer, I was mistaken then (or perhaps .NET Reflector had it). Either way, #3082 is my take on it.
